frontend/app/src/components/trades/consts.ts

```diff
@@ -0,0 +1,28 @@
+import { TradeLocationData } from '@/components/trades/type';
+
+export const tradeLocations: TradeLocationData[] = [
```

Comment:
There's still missing valid locations here, no?

Reply:
well, only potentially valid in the future.

Reply:
Huh? What about coinbase, coinbasepro, etc?

Reply:
You were probably looking at the wrong commit.